src/clarity/wizard/wizard.ts

```diff
@@ -124,6 +128,16 @@ export class Wizard extends Tabs {
         this._openChanged.emit(false);
     }
 
+    // _close --
+    //
+    // This is a private function that is called on the click of the close / cancel
+    // button and emits the onCancel event of the active tab.
+    _close(): void {
```

Comment:
This isn't called when the user presses the Escape key, even though it do so closes the wizard. Can you add a listener for it? You can check out how [the modal](https://github.com/vmware/clarity/blob/master/src/clarity/modal/modal.ts#L78)  does it for instance.